docker/edxapp/vars.yml

```diff
@@ -22,9 +22,12 @@ EDXAPP_MYSQL_PASSWORD_MIGRATE: $EDXAPP_MYSQL_PASSWORD
 
 # Mongo DB
 EDXAPP_MONGO_HOSTS: ['mongodb']
-EDXAPP_MONGO_PORT: 27017
+EDXAPP_MONGO_PORT: "27017"
 EDXAPP_MONGO_DB_NAME: 'docker-edxapp'
 EDXAPP_MONGO_USER: 'edxapp'
 EDXAPP_MONGO_PASSWORD: 'daheiYae2c'
 MONGO_BIND_IP: '0.0.0.0'
-mongo_create_users: true
+
+# Forum
+EDXAPP_COMMENTS_SERVICE_URL:  'http://forum:18080'
```

Comment:
Just noting (for the future) that this only works when running through Fig. We might want to get the forum URL from the docker link env variables.